src/CompanyService.Business/GetDepartmentByIdCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LT.DigitalOffice.CompanyService.Business.Interfaces;
+using LT.DigitalOffice.CompanyService.Data.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Dto.Models;
+using System;
+using Microsoft.AspNetCore.Mvc;
+using LT.DigitalOffice.CompanyService.Mappers.ResponsesMappers.Interfaces;
+using Microsoft.Extensions.Logging;
+
+namespace LT.DigitalOffice.CompanyService.Business
+{
+    public class GetDepartmentByIdCommand : IGetDepartmentByIdCommand
+    {
+        private readonly IDepartmentRepository repository;
+        private readonly IDepartmentMapper mapper;
+        private ILogger<GetDepartmentByIdCommand> _logger;
+
+        public GetDepartmentByIdCommand(
+            [FromServices] ILogger<GetDepartmentByIdCommand> logger,
+            [FromServices] IDepartmentRepository repository,
+            [FromServices] IDepartmentMapper mapper)
+        {
+            _logger = logger;
+            this.repository = repository;
+            this.mapper = mapper;
```

Comment:
I think it would be better if all three variables start with "_" or without "_" and use "this" in the constructor.

Reply:
done